nautobot/extras/tests/test_approvals.py

```diff
@@ -25,26 +27,35 @@ def setUpTestData(cls):
         cls.approval_workflow_def_1 = models.ApprovalWorkflowDefinition.objects.create(
             name="Test Approval Workflow Definition 1",
             model_content_type=job_ct,
```

Comment:
Need to change all of these to use `scheduled_job_ct` if we're no longer having `job_ct` support approval workflows?